web/client/plugins/tutorial/preset/geostory_edit_tutorial.js

```diff
@@ -82,16 +46,16 @@ module.exports = [
                 withSave: false
             }
         }
-    }, {
-        translation: "geostoryEditContainer",
+    }, */ {
+        translationHTML: "geostoryEditContainer",
         selector: ".ms-geostory",
-        position: "auto",
+        position: "auto"/* ,
         action: {
             next: {
                 type: 'GEOSTORY:TOGGLE_SETTINGS_PANEL',
                 withSave: false
             }
-        }
+        }*/
```

Comment:
Some steps are commented, do we still need those?

Reply:
yes, going to restore them